test/Integration/hoist-vector-transfer-brgemm.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: tpp-opt %s  | tpp-run -e entry --entry-point-result=void  -print > %t.1
+// RUN: tpp-opt %s --loop-invariant-code-motion  --vectorization-pass --loop-invariant-code-motion --hoist-vector-transfer | tpp-run -e entry --entry-point-result=void  -print > %t.2
```

Comment:
Add a new RUN line that prints the IR and make sure that the contract read/write gets hoisted. Otherwise we won't know if your pass actually did anything, and the result will be the same.

Reply:
Same. The check is already done in `unit` tests.